SystemInfo/Collector/JsonComposerLockSystemInfoCollector.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * File containing the JsonComposerLockSystemInfoCollector class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\SystemInfo\Collector;
+
+use EzSystems\EzSupportToolsBundle\SystemInfo\Value;
+
+/**
+ * Collects information about installed Composer packages, by reading json from composer.lock.
+ */
+class JsonComposerLockSystemInfoCollector implements SystemInfoCollector
+{
+    /**
+     * @var string Installation root directory
+     */
+    private $installDir;
+
+    public function __construct($installDir)
```

Comment:
Why did you choose to use the directory as the argument, as opposed to the complete path to a lock file ?